packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -383,13 +383,15 @@ class FlutterDriver {
   final int _driverId;
 
   /// Client connected to the Dart VM running the Flutter application
-  final VMServiceClient _serviceClient;
+  final VMServiceClient serviceClient;
```

Comment:
Can you document why one might want to use this?

Reply:
Done.